spec.md

```diff
@@ -395,7 +395,7 @@ For success responses, a broker may return the following fields. For error respo
 
 | Response field  |  Type | Description  |
 |---|---|---|
-|  dashboard_url | string  |  The URL of a web-based management user interface for the service instance; we refer to this as a service dashboard. The URL should contain enough information for the dashboard to identify the resource being accessed (<code>9189kdfsk0vfnku</code> in the example below). |
+|  dashboard_url | string  |  The URL of a web-based management user interface for the service instance; we refer to this as a service dashboard. The URL should contain enough information for the dashboard to identify the resource being accessed (<code>9189kdfsk0vfnku</code> in the example below).  Note: a broker that wishes to return `dashboard_url` for a service instance must return it with the initial response to the provision request, even if the service is provisioned asynchronously. |
```

Comment:
Since I'm not sure which PR will get merged first, can you s/must/MUST/ so we don't miss it.